### PR TITLE
[4.2] Tinymce 5.10.5

### PR DESCRIPTION
**Pull request by brianteeman**
Update to the latest release in the 5 series.

https://www.tiny.cloud/docs/release-notes/release-notes5105/

### CODE REVIEW

**Comment by brianteeman** [comment]
I have no idea why drone is failing. There were no problems at all on the branch this was submitted against

**Comment by HLeithner** [comment]
> I have no idea why drone is failing. There were no problems at all on the branch this was submitted against

drone restarted and solved

File: plugins/editors/tinymce/tinymce.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <extension type="plugin" group="editors" method="upgrade">
 	<name>plg_editors_tinymce</name>
-	<version>5.10.3</version>
-	<creationDate>2005-08</creationDate>
+	<version>5.10.5</version>
+	<creationDate>2005-2021</creationDate>
```
Comment (laoneo) [inline plugins/editors/tinymce/tinymce.xml]: This here should be reverted, otherwise it would not be sortable. More information can be found in #27162.

Comment (brianteeman) [reply plugins/editors/tinymce/tinymce.xml]: This was caused by @roland-d merging in 4.2. 
The code was correct when it was submitted

**Comment by roland-d** [comment]
Thank you

**Pull request merged**